styled-components.js

```diff
@@ -11,12 +11,6 @@ module.exports = {
     // OK
     // background-image: url(${toStaticUrl('/img/foo.png')});
     "function-url-quotes": null,
-    // This rule cannot recognize CSS functions in template literals
-    // Error
-    // transform: ${({ rotate }) => `rotate(${rotate}deg)`};
-    // OK
-    // transform: ${({ rotate }) => `rotate(${rotate}deg) `};
-    "function-whitespace-after": null,
```

Comment:
![memo](https://img.shields.io/badge/review-memo-white.svg)
`function-whitespace-after` is now deprecated and removed from `stylelint-config-standard`.

https://github.com/stylelint/stylelint-config-standard/compare/29.0.0...30.0.0#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L93